src/main/java/com/demo/crud/operation/controller/HomeController.java

```diff
@@ -0,0 +1,84 @@
+package com.demo.crud.operation.controller;
+
+import com.demo.crud.operation.beans.Response;
+import com.demo.crud.operation.beans.UserDetails;
+import com.demo.crud.operation.service.DataService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ * @author Amit Ojha
+ */
+@RestController
+@RequestMapping("/demo-crud")
+@Api(tags = "User-Controller", description = "User Controller")
+public class HomeController {
```

Comment:
Name should be related to the functionality like UserController not HomeController.